server/utils/slices.go

```diff
@@ -0,0 +1,12 @@
+package utils
+
+// SlicesContains reports whether v is present in s.
+// https://pkg.go.dev/golang.org/x/exp/slices#Contains
```

Comment:
this might be available go1.21 or later.

https://github.com/golang/go/issues/57433